### PR TITLE
DE4254 - Group meeting time issues

### PR DESCRIPTION
**Pull request by JoeKer1**
We are not adjusting time's based on timezone so whatever time is stored in MP will display on the group no matter where you're at in the world. After this PR. No more magic 3 minutes.

### CODE REVIEW

File: src/app/components/create-group/preview/create-group-preview.component.ts [hunk 1]
```diff
@@ -75,7 +73,7 @@ export class CreateGroupPreviewComponent implements OnInit {
       )
         .subscribe((returnData) => {
           this.toastr.success('Successfully edited group!');
-          this.smallGroupPin.gathering.meetingTime = this.timeHelperService.convertTime(this.smallGroupPin.gathering.meetingTime);
+          // this.smallGroupPin.gathering.meetingTime = this.timeHelperService.convertTime(this.smallGroupPin.gathering.meetingTime);
```
Comment (plachmann) [inline src/app/components/create-group/preview/create-group-preview.component.ts]: Commented Residue from the timeHelperService.

File: src/app/pipes/utc-time-format.pipe.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    const datewithouttimezone = moment.utc([dateValue.getUTCFullYear(),
+      dateValue.getUTCMonth(),
+      dateValue.getUTCDate(),
+      dateValue.getUTCHours(),
+      dateValue.getUTCMinutes(),
+      dateValue.getUTCSeconds()]);
+    return datewithouttimezone.format('h:mm a');
+  }
+
+}
```
Comment (plachmann) [inline src/app/pipes/utc-time-format.pipe.ts]: I like it.

File: src/app/services/time-helper.service.ts [hunk 3]
```diff
@@ -1,116 +0,0 @@
...
-  }
-
-  private addLeadingZeroIfMissing(hours: string): string{
-    if(hours.length < 2){
-      hours = `0${hours}`;
-    }
-    return hours;
-  }
-
-}
```
Comment (plachmann) [inline src/app/services/time-helper.service.ts]: Goodbye timeHelperService. You will not be missed.

File: src/app/components/create-group/preview/create-group-preview.component.ts (hunk 1 above)
Comment (JoeKer1) [reply src/app/components/create-group/preview/create-group-preview.component.ts]: ugh i had that change and forgot to commit it lol

**Pull request merged**